src/Symfony/Bundle/FrameworkBundle/composer.json

```diff
@@ -79,7 +79,8 @@
         "symfony/translation": "<4.3",
         "symfony/twig-bridge": "<4.1.1",
         "symfony/validator": "<4.1",
-        "symfony/workflow": "<4.3"
+        "symfony/workflow": "<4.3",
+        "symfony/http-kernel": ">=5.0"
```

Comment:
this is needed only when the dep is in "require-dev"
here, it's in "require", so this should be removed as a duplicate.
Please check the other changes for similar situations.

Reply:
I have removed the conflict section for all of them because they don't require it in dev